Just extensions (#987)
radiantearth/stac-spec: extensions/README.md

This folder contains extensions to the SpatioTemporal Asset Catalog specification. The core STAC specification 
defines only a minimal core, but is designed for extension. It is expected that most real-world
implementations will use several extensions to fully describe their data.  

Extensions to the core STAC specification provide additional JSON fields that can be used to better describe

> the data. Most tend to be about describing a particular data type or domain, but some bring in additional

Reviewer: ```suggestion
the data. Most tend to be about describing a particular domain or type of data, but some imply
```

functionality. 

> Extensions should include narrative explaining the fields, a comprehensive example, and a JSON-Schema to validate compliance.

Reviewer: ```suggestion
Extensions should include a JSON Schema precisely describing the structure, a natural language description of the fields, and thorough examples.
```

### Use of arrays and objects

> For extensions it is recommended to use use arrays only as true (potentially sorted) enumerations/lists without having additional meaning and to avoid objects whenever possible.

Reviewer: ```suggestion
For extensions, it is recommended to 

1. Use arrays only as enumerations/lists (possibly sorted), without implying additional meaning (such as order)
2. To avoid using nested objects, in favor of multiple attributes with a similar naming scheme.
```
